sessionctx/context.go

```diff
@@ -135,7 +135,8 @@ type Context interface {
 
 	// StmtCommit flush all changes by the statement to the underlying transaction.
 	StmtCommit()
-	// StmtRollback provides statement level rollback.
+	// StmtRollback provides statement level rollback. The parameter `forPessimisticRetry` should be true iff it's used
```

Comment:
Where is the `forPessimisticRetry` parameter?

Reply:
Ah sorry, it was removed and I forgot to change back the comment